tests/zfs-tests/tests/functional/rsend/rsend.kshlib

```diff
@@ -121,6 +121,7 @@ function cleanup_pool
 	log_must rm -rf $BACKDIR/*
 
 	if is_global_zone ; then
+		log_must df >/dev/null
```

Comment:
Is this some magic?

Reply:
It's terrible, terrible black magic.  I'm not sure why it avoids the issue, but it does.

Reply:
See #12898 for the reason this works.  I've used that PR to add a comment here which references the root issue, at least until it's properly addressed.